Rspec/lib/sol_ver.rb

```diff
@@ -0,0 +1,21 @@
+class Solver
+  def factorial(num)
+    return 'Error! number is negative.' if num.negative?
+
+    result = 1
+    (1..num).each { |i| result *= i }
+    result
+  end
+
+  def reverse(str)
+    str.reverse
+  end
+
+  def fizzbuzz(num)
+    return 'fizzbuzz' if (num % 15).zero?
+    return 'fizz' if (num % 3).zero?
+    return 'buzz' if (num % 5).zero?
+
+    num
+  end
```

Comment:
- [ ]  As per the project requirement the commit history sequence should clearly show that each of the Solver class methods was tested before writing the code, that's not the case with your code, kindly make sure to create the test cases before writing any code so your project follow the TDD method

Reply:
> * [ ]   As per the project requirement the commit history sequence should clearly show that each of the Solver class methods was tested before writing the code, that's not the case with your code, kindly make sure to create the test cases before writing any code so your project follow the TDD method

Thank you @Ismailco for your feed back. Howerver, it seems you did not follow the commit order, otherwise you would see that all test cases were written before corresponding methods were added. Please check again!

Reply:
@ernestmusong, sorry for this the way you committed your changes one by one has me confused, I apologise for my mistake